.github/workflows/build_master.yaml

```diff
@@ -29,6 +29,7 @@ jobs:
           git config user.email "teamfamilie@nav.no"
           git config user.name "Team familie"
           git remote set-url origin https://x-access-token:${GITHUB_TOKEN}@github.com/${GITHUB_REPOSITORY}.git
+          git checkout yarn.lock package.json
```

Comment:
What is this? Seems a bit hacky..

Reply:
Lerna has an issue that it may cause build error when yarn.lock is changed and committed.
I googled a lot and found this walk around. I have fixed the branch build script before, but the master build script was not fixed.